### PR TITLE
Apply changes auto done by pde to target files.

### PR DESCRIPTION
**Pull request by akurtakov**

### CODE REVIEW

**Comment by akurtakov** [comment]
@laeubi as pde changes these I thought it's good idea to push the changes but tests are failing. What should we do here?

**Comment by laeubi** [comment]
I would not change existing integration tests as these also test legacy behavior for backward compatibilty to old format.
Also if you update make sure to use the latest m2e-snapshot as some never features are not present in the current release.

**Comment by akurtakov** [comment]
> I would not change existing integration tests as these also test legacy behavior for backward compatibilty to old format. Also if you update make sure to use the latest m2e-snapshot as some never features are not present in the current release.

Problem with keeping them is that if one just opens the target platform page these files are changed and one has to manually revert changes in them.

**Comment by laeubi** [comment]
I know, but it should not be necessary to open them at all I think. Don't know if we can mark them to ignore changes with some git property?

**Comment by laeubi** [comment]
BTW: the changes in `tycho-bundles-target.target` should be fine and the only one to work with on a regular basis.

File: tycho-its/projects/target.maven.autofeature/test.target [hunk 1]
```diff
@@ -17,10 +17,6 @@
 			</dependencies>
 		</location>
 		<location includeSource="true" missingManifest="generate" type="Maven">
-
```
Comment (laeubi) [inline tycho-its/projects/target.maven.autofeature/test.target:20]: This change seems to indicate that not the latest m2e-snapshot is used here and what makes the test fail

File: tycho-bundles/org.eclipse.tycho.p2.resolver.impl.test/resources/targetresolver/mavenDep.target [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <?pde version="3.8"?>
 <target name="mavenDep.targe">
 	<locations>
-		<location includeDependencyDepth="none" includeDependencyScope="compile" includeSource="false" missingManifest="ignore" type="Maven">
+		<location includeDependencyScope="compile" missingManifest="ignore" type="Maven">
```
Comment (laeubi) [inline tycho-bundles/org.eclipse.tycho.p2.resolver.impl.test/resources/targetresolver/mavenDep.target:5]: This removes the latest changes from m2e so maybe another reason why the test fails.

**Review by laeubi — CHANGES_REQUESTED** [review]
I would propose that we only change the tycho-bundles.target file and leave the other ones alone.
These other targets are essentially test-resources and should never be touched/modified.

**Comment by akurtakov** [comment]
OK, I'm giving up on this one as I clearly haven't followed deep enough and don't have latest snapshots of everything.

**Pull request closed without merging**